amgut/handlers/human_survey.py

```diff
@@ -1,15 +1,74 @@
+from amgut import media_locale, text_locale
 from amgut.connections import ag_data, redis
 from amgut.handlers.base_handlers import BaseHandler
 from amgut.lib.util import store_survey
 from amgut.lib.survey_supp import primary_human_survey
+from amgut.lib.mail import send_email
+
+
+def build_consent_form(consent_info):
+    tl = text_locale['new_participant.html']
+    # email out the consent form
+    if consent_info['age_range'] == '0-6':
+        message = ("%s<p>%s: %s</p><p>%s: %s</p><p>%s: %s</p>"
+                   "<p>%s: %s</p><p>%s: %s</p><p>%s: %s</p>") %\
+            (tl['CONSENT_YOUR_CHILD'],
+             tl['PARTICIPANT_NAME'], consent_info['participant_name'],
+             tl['PARTICIPANT_EMAIL'], consent_info['participant_email'],
+             tl['PARTICIPANT_PARENT_1'], consent_info['parent_1_name'],
+             tl['PARTICIPANT_PARENT_2'], consent_info['parent_2_name'],
+             tl['PARTICIPANT_DECEASED_PARENTS'],
+             consent_info['deceased_parent'],
+             tl['DATE_SIGNED'], str(consent_info['date_signed']))
+
+    elif consent_info['age_range'] == '7-12':
+        message = ("%s<p>%s: %s</p><p>%s: %s</p><p>%s: %s</p>"
+                   "%s<p>%s: %s</p><p>%s: %s</p><p>%s: %s</p>"
+                   "<p>%s: %s</p>") %\
+            (tl['ASSENT_7_12'],
+             tl['PARTICIPANT_NAME'], consent_info['participant_name'],
+             tl['PARTICIPANT_EMAIL'], consent_info['participant_email'],
+             tl['OBTAINER_NAME'], consent_info['assent_obtainer'],
+             tl['CONSENT_YOUR_CHILD'],
+             tl['PARTICIPANT_PARENT_1'], consent_info['parent_1_name'],
+             tl['PARTICIPANT_PARENT_2'], consent_info['parent_2_name'],
+             tl['PARTICIPANT_DECEASED_PARENTS'],
+             consent_info['deceased_parent'],
+             tl['DATE_SIGNED'], str(consent_info['date_signed']))
+
+    elif consent_info['age_range'] == '13-17':
+        message = ("%s<p>%s: %s</p><p>%s: %s</p>"
+                   "%s<p>%s: %s</p><p>%s: %s</p><p>%s: %s</p>"
+                   "<p>%s: %s</p>") %\
+            (tl['ASSENT_13_17'],
+             tl['PARTICIPANT_NAME'], consent_info['participant_name'],
+             tl['PARTICIPANT_EMAIL'], consent_info['participant_email'],
+             tl['CONSENT_YOUR_CHILD'],
+             tl['PARTICIPANT_PARENT_1'], consent_info['parent_1_name'],
+             tl['PARTICIPANT_PARENT_2'], consent_info['parent_2_name'],
+             tl['PARTICIPANT_DECEASED_PARENTS'],
+             consent_info['deceased_parent'],
+             tl['DATE_SIGNED'], str(consent_info['date_signed']))
+
+    elif consent_info['age_range'] == '18-plus':
+        message = "%s<p>%s: %s</p><p>%s: %s</p><p>%s: %s</p>" %\
+            (tl['CONSENT_18'],
+             tl['PARTICIPANT_NAME'], consent_info['participant_name'],
+             tl['PARTICIPANT_EMAIL'], consent_info['participant_email'],
+             tl['DATE_SIGNED'], str(consent_info['date_signed']))
+
+    else:
```

Comment:
Yeah, I have it there as a catch-all to figure out if things are going wrong, like in the case of catching the multi-email issue.